projects/demo/src/pages/recipes/placeholder/examples/3-date/component.ts

```diff
@@ -28,22 +26,6 @@ export class PlaceholderDocExample3 {
     inputRef!: ElementRef<HTMLInputElement>;
 
     readonly maskitoOptions = mask;
-    value = '';
-
-    onBlur(): void {
-        this.value = removePlaceholder(this.value);
-    }
-
-    onFocus(): void {
-        const initialValue = this.value;
```

Comment:
This example becomes simple 🚀 

Drop this line to enable Stackblitz and addition of default tabs:
https://github.com/Tinkoff/maskito/blob/36a88fd704ad018b07569b9497263d750a56aaba/projects/demo/src/pages/recipes/placeholder/placeholder-doc.component.ts#L28